compiler/src/dotty/tools/dotc/reporting/messages.scala

```diff
@@ -325,11 +325,11 @@ import transform.SymUtils._
 
       // A list of possible candidate strings with their Levenstein distances
       // to the name of the missing member
-      def closest: List[(Int, String)] = candidates
+      def closest: List[(Int, Symbol)] = candidates
         .toList
-        .map(n => (distance(n, missing), n))
-        .filter((d, n) => d <= maxDist && d < missing.length && d < n.length)
-        .sorted  // sort by distance first, alphabetically second
+        .map(n => (distance(n.name.show, missing), n))
```

Comment:
For less confusion I would advise to rename `n` to `s` all around here as we're now dealing with symbols instead of names

Reply:
I would use `sym` here (as in `val candidates = ...`) !